runtime/src/accounts_db.rs

```diff
@@ -3925,41 +3945,43 @@ impl AccountsDB {
     fn scan_snapshot_stores(
         storage: &[SnapshotStorage],
         simple_capitalization_enabled: bool,
-    ) -> (Vec<Vec<CalculateHashIntermediate>>, Measure) {
+    ) -> (Vec<Vec<CalculateHashIntermediate>>, Measure, usize, Measure) {
```

Comment:
Passing around `Measure` is getting awkward, we have a somewhat standard way of accumulating metrics into mutable stats struct, can we match that pattern so it looks like the rest of the code?

Reply:
CleanKeyTimings is one example:
https://github.com/solana-labs/solana/blob/ab0f4c69aaea10fa62b887d3212d0b09c026c0f9/runtime/src/accounts_db.rs#L1321-L1324
or
ShrinkStats:
https://github.com/solana-labs/solana/blob/ab0f4c69aaea10fa62b887d3212d0b09c026c0f9/runtime/src/accounts_db.rs#L874-L890